.env.example

```
@@ -1,37 +1,109 @@
+# Default: development
+# APP_ENV=development
+
+# Optional: allow OAuth over HTTP in development (disable in production)
+# OAUTHLIB_INSECURE_TRANSPORT=1
+
+# -----------------------------
+# FalkorDB connection (REQUIRED / preferred)
+# -----------------------------
+# Preferred: single connection URL. Edit to point at your FalkorDB/Redis instance.
+# Example: redis://localhost:6379/0
+FALKORDB_URL=redis://localhost:6379/0  # REQUIRED - change to your FalkorDB URL
```

dotenv-linter (3.3.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)